cli/JSXV4.md

```diff
@@ -18,16 +18,26 @@ To build an entire project in V4 mode, including all its dependencies, use the n
 ### Dependency-level config
 
 Dependencies inherit the `jsx` configuration of the root project. So if the root project uses V4 then the dependencies are built using V4, and the same for V3.
```

Comment:
Is it still correct?

Reply:
Yes

Reply:
I confirm that it is correct. I found that the error was caused by `bsc-flags` with `-open ReactV3`missing.

Reply:
I considered adding `"-open ReactV3"` into `bsc-flags` automatically under configuration `jsx.version=3` for seconds, but the premise is needed that the latest rescript-react should be one of the dependencies.
Yeap, it is too much.

Reply:
I'm clear now.